manifests/config.pp

```diff
@@ -30,6 +30,14 @@
   pulpcore::admin { 'collectstatic --noinput':
     refreshonly => true,
     subscribe   => Concat['pulpcore settings'],
+    require     => File[$pulpcore::pulp_static_root],
+  }
+
+  file { '/usr/libexec/pulpcore':
```

Comment:
Might be worth turning this path into an internal variable due to it's usage throughout.

Reply:
I also thought about that but didn't manage to update it before the weekend started. Will update.